be/src/olap/tablet_meta.h

```diff
@@ -155,7 +155,7 @@ class TabletMeta {
     inline const bool in_restore_mode() const;
     inline OLAPStatus set_in_restore_mode(bool in_restore_mode);
 
-    inline const TabletSchema& tablet_schema() const;
+    inline TabletSchema& tablet_schema();
```

Comment:
Prefer `mutable_tablet_schema()` to return a pointer.
Normally, const modification is required when returning a reference

Reply:
OK